package.json

```diff
@@ -11,50 +11,51 @@
     "@apollo/react-hooks": "^3.1.4",
     "@expo/vector-icons": "^10.0.0",
     "@react-native-community/art": "^1.2.0",
-    "@react-native-community/masked-view": "^0.1.7",
+    "@react-native-community/datetimepicker": "2.2.2",
+    "@react-native-community/masked-view": "0.1.6",
     "apollo-boost": "^0.4.7",
     "apollo-link-context": "^1.0.20",
     "apollo-link-state": "^0.4.2",
     "axios": "^0.19.2",
     "dotenv": "^8.2.0",
-    "expo": "^36.0.2",
-    "expo-blur": "^8.1.0",
-    "expo-haptics": "^8.1.0",
-    "expo-permissions": "^8.1.0",
+    "expo": "^37.0.0",
+    "expo-blur": "~8.1.0",
+    "expo-haptics": "~8.1.0",
+    "expo-permissions": "~8.1.0",
     "graphql": "^15.0.0",
     "install": "^0.13.0",
     "lodash": "^4.17.15",
     "moment": "^2.24.0",
     "native-base": "^2.13.8",
-    "react": "~16.9.0",
-    "react-dom": "~16.9.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz",
+    "react": "16.9.0",
+    "react-dom": "16.9.0",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
     "react-native-chart-kit": "^5.6.0",
     "react-native-confetti-cannon": "^1.5.0",
     "react-native-dotenv": "^0.2.0",
     "react-native-drawer": "^2.5.1",
     "react-native-elements": "^1.2.7",
     "react-native-fontawesome": "^7.0.0",
-    "react-native-gesture-handler": "^1.6.0",
+    "react-native-gesture-handler": "~1.6.0",
     "react-native-material-dropdown": "^0.11.1",
     "react-native-neomorph-shadows": "^1.0.3",
     "react-native-paper": "^3.8.0",
     "react-native-progress-circle": "^2.1.0",
     "react-native-really-awesome-button": "^1.6.0",
-    "react-native-reanimated": "^1.7.0",
-    "react-native-safe-area-context": "^0.7.3",
-    "react-native-screens": "^2.2.0",
-    "react-native-svg": "9.13.3",
+    "react-native-reanimated": "~1.7.0",
+    "react-native-safe-area-context": "0.7.3",
+    "react-native-screens": "~2.2.0",
+    "react-native-svg": "11.0.1",
     "react-native-swipe-list-view": "^3.0.0",
-    "react-native-web": "~0.11.7",
+    "react-native-web": "^0.11.7",
     "react-navigation": "^4.2.2",
     "react-navigation-stack": "^2.2.3",
     "react-navigation-tabs": "^2.8.2",
     "victory-native": "^34.1.0"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
-    "babel-preset-expo": "~8.0.0",
+    "babel-preset-expo": "^8.1.0",
```

Comment:
@ChrisCruze this change might have tangible effects down the road. Was this required to be done by a new library we installed?